src/VisualStudio/Core/SolutionExplorerShim/HierarchyItemToProjectIdMap.cs

```diff
@@ -47,6 +46,16 @@ public bool TryGetProjectId(IVsHierarchyItem hierarchyItem, string targetFramewo
             var project = _workspace.DeferredState.ProjectTracker.ImmutableProjects
                     .Where(p =>
                     {
+                        // We're about to access various properties of the IVsHierarchy associated with the project.
```

Comment:
They do, but with a different implementation, and project system support is a WIP for them. Right now they aren't using this code path and this change won't affect them. I talked with @KevinRansom and we'll make changes here to explicitly support F# if and when necessary.